completions/zsh/_crio

```diff
@@ -7,7 +7,7 @@ it later with **--config**. Global options will modify the output.' 'version:dis
+  opts=('--absent-mount-sources-to-reject' '--additional-devices' '--address' '--allowed-devices' '--apparmor-profile' '--big-files-temporary-dir' '--bind-mount-prefix' '--blockio-config-file' '--cdi-spec-dirs' '--cgroup-manager' '--clean-shutdown-file' '--cni-config-dir' '--cni-default-network' '--cni-plugin-dir' '--config' '--config-dir' '--conmon' '--conmon-cgroup' '--conmon-env' '--container-attach-socket-dir' '--container-exits-dir' '--ctr-stop-timeout' '--decryption-keys-path' '--default-capabilities' '--default-env' '--default-mounts-file' '--default-runtime' '--default-sysctls' '--default-transport' '--default-ulimits' '--device-ownership-from-security-context' '--drop-infra-ctr' '--enable-metrics' '--enable-profile-unix-socket' '--enable-tracing' '--gid-mappings' '--global-auth-file' '--grpc-max-recv-msg-size' '--grpc-max-send-msg-size' '--hooks-dir' '--image-volumes' '--infra-ctr-cpuset' '--insecure-registry' '--internal-wipe' '--irqbalance-config-file' '--listen' '--log' '--log-dir' '--log-filter' '--log-format' '--log-journald' '--log-level' '--log-size-max' '--metrics-cert' '--metrics-collectors' '--metrics-key' '--metrics-port' '--metrics-socket' '--minimum-mappable-gid' '--minimum-mappable-uid' '--namespaces-dir' '--no-pivot' '--pause-command' '--pause-image' '--pause-image-auth-file' '--pids-limit' '--pinns-path' '--profile' '--profile-cpu' '--profile-mem' '--profile-port' '--rdt-config-file' '--read-only' '--registries-conf' '--registries-conf-dir' '--registry' '--root' '--runroot' '--runtimes' '--seccomp-profile' '--seccomp-use-default-when-empty' '--selinux' '--separate-pull-cgroup' '--signature-policy' '--stats-collection-period' '--storage-driver' '--storage-opt' '--stream-address' '--stream-enable-tls' '--stream-idle-timeout' '--stream-port' '--stream-tls-ca' '--stream-tls-cert' '--stream-tls-key' '--tracing-endpoint' '--tracing-sampling-rate-per-million' '--uid-mappings' '--version-file' '--version-file-persist' '--help' '--version')
```

Comment:
@klihub, not on you and no action required from you, but we need to break this in a way the diff is easier to read.
That would be a good first issue for a newcomer. :-)

Reply:
@fidencio I tried a stab at this with this [PR](https://github.com/cri-o/cri-o/pull/5586).

Since it looked to me as if zsh completions were generated broken since [d546893397](https://github.com/cri-o/cri-o/blob/62cf7fbcee4a19b8ea59d8e4fdf0803fc15a3cdc/cmd/crio/wipe.go#L19) with `'` present within `'`-quoted strings, so I tried to fix that, too.

WDYT?